.vscode/tasks.json

```diff
@@ -0,0 +1,10 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"label": "build",
+			"command": "msbuild",
+			"args": ["-t:build", "-p:Configuration=Release", "-verbosity:m", "-nologo"]
```

Comment:
```suggestion
			"args": ["-t:build", "-p:Configuration=Release", "-verbosity:m", "-nologo"],
			"problemMatcher": ["$msCompile"]
```

Reply:
It reports compiler errors in the problems pane:

![image](https://user-images.githubusercontent.com/756669/85945349-78db4100-b93d-11ea-9aa1-d4698c259a52.png)

However it is redundant to the Roslyn Analyzers so it duplicates all errors and the paths don't work when click on them.